template/src/app-state/useVirtualBackground.native.ts

```diff
@@ -0,0 +1,167 @@
+import {useEffect, useRef} from 'react';
+import {
+  Option,
+  useVB,
+  type VBMode,
+} from '../../src/components/virtual-background/useVB';
+import RNFS from 'react-native-fs';
+import resolveAssetSource from 'react-native/Libraries/Image/resolveAssetSource';
+import {useSidePanel} from '../utils/useSidePanel';
+import {SidePanelType} from '../subComponents/SidePanelEnum';
+
+export interface virtualBackgroundInterface {
+  vbOptions: Option[]; // options available in vb panel
+  setVBOptions: (options: Option[]) => void;
+  saveVBOption: (type: VBMode, path: string) => void;
+  applyVBOption: () => void;
```

Comment:
setVBOptions, saveVBOptions, applyVBOption is getting a little confusing, as all pointing to a similar thing.

update to virtualBackgrounds, addNewVirtualBackground, setVBPreview, applyVirtualBackground

Reply:
done